blocks/collapsible/collapsible.css

```diff
@@ -0,0 +1,87 @@
+:root {
+    /* font color */
+    --mt-collapsible-question-font-color: var(--mt-global-color-base-primary);
+    --mt-collapsible-question-expanded-font-color: var(--mt-global-color-accent-quinary);
+
+    /* font weight */
+    --mt-collapsible-question-hover-font-weight: var(--mt-font-weight-boldest);
+
+    /* line height */
+    --mt-collapsible-question-line-height: 1rem;
+    --mt-collapsible-answer-line-height: 1.5rem;
+
+    /* border color */
+    --mt-collapsible-answer-border-color: #d5d8dc;
```

Comment:
Please use `var(--mt-global-color-line-grey)` instead of hex.

@solaris007 Are we adding css vars into the `block.css` or `styles.css`?

Reply:
if the vars are not used anywhere else, but still serve a readability and re-use purpose, it is totally ok to have them in the block's CSS.

Reply:
@solaris007 to me, it makes more sense to separate into block.css...some of us put CSS vars into styles.css already...what is the plan to make this consistent?